packages/block-editor/src/components/preview-options/README.md

```diff
@@ -0,0 +1,63 @@
+# Preview Options
+
+The `PreviewOptions` component displays the list of different preview options available in the editor.
+
+It returns a [`DropdownMenu`](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown-menu) component with these different options. The options currently available in the editor are Desktop, Mobile, Tablet and "Preview in new tab".
+
+![Preview options dropdown menu](https://make.wordpress.org/core/files/2020/09/preview-options-dropdown-menu.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders the previews options of the editor in a dropdown menu.
+
+```jsx
+import { PreviewOptions } from '@wordpress/block-editor';
+
+const MyPreviewOptions = () => (
+    <PreviewOptions
+        isEnabled={ true }
+        className="edit-post-post-preview-dropdown"
+        deviceType={ deviceType }
+        setDeviceType={ setPreviewDeviceType }
+    >
+);
+```
+
+### Props
```

Comment:
Might be good to also add to each prop whether it is Required; in this case only `deviceType` and `setDeviceType` are required props; the rest are optional.